### PR TITLE
Added task id command and integration test

### PR DESCRIPTION
**Pull request by zhanggbj**
1.Added task id command
2.Added one simple integration test

[#115478103](https://www.pivotaltracker.com/story/show/115478103)

### CODE REVIEW

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,

This command is till WIP and need to add more integration test, but I would like to submit it for review now, so issues could be exposed early. I will continue working with this command. 
I know there is still existed PR with unaddressed comments. I haven't made up my mind how to update it. I'll fix them later.

Thank you for your review:)

File: cmds/bmp/task_command.go [hunk 1]
```diff
@@ -46,10 +49,34 @@ func (cmd taskCommand) Options() cmds.Options {
...
+
 	return true, nil
 }
 
 func (cmd taskCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s comamnd: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
+
+	level := "event"
+	if cmd.options.Debug == true {
+		level = "deubg"
```
Comment (maximilien) [inline cmds/bmp/task_command.go]: Typo, spelling: this should be `debug`

File: cmds/bmp/task_command_test.go [hunk 2]
```diff
@@ -69,13 +72,92 @@ var _ = Describe("task command", func() {
...
+		Context("validates a bad TaskCommand", func() {
+			BeforeEach(func() {
+				options = cmds.Options{
+					Verbose: false,
+					TaskID:  0,
+					Debug:   false,
+				}
+			})
+
+			It("no task_id", func() {
```
Comment (maximilien) [inline cmds/bmp/task_command_test.go]: change description to something like: `"fails when there are no task_id or task_id is 0"`

File: cmds/bmp/task_command_test.go [hunk 3]
```diff
@@ -69,13 +72,92 @@ var _ = Describe("task command", func() {
...
+		})
 	})
 
 	Describe("#Execute", func() {
-		It("executes a good TaskCommand", func() {
-			rc, err := cmd.Execute(args)
-			Expect(rc).To(Equal(0))
-			Expect(err).ToNot(HaveOccurred())
+		Context("executes a good TaskCommand", func() {
+			Context("executes TaskCommand with default event level", func() {
```
Comment (maximilien) [inline cmds/bmp/task_command_test.go]: Typically inner Context descriptions start with `when` so this one would be:

```
Context("when a default event level is passed", ...)
```

File: cmds/bmp/task_command_test.go [hunk 4]
```diff
@@ -69,13 +72,92 @@ var _ = Describe("task command", func() {
...
+					fakeBmpClient.TaskOutputResponse.Status = 200
+					fakeBmpClient.TaskOutputErr = nil
+					options = cmds.Options{
+						Verbose: false,
+						TaskID:  1,
+						Debug:   false,
+					}
+				})
+
+				It("executes a good TaskCommand ", func() {
```
Comment (maximilien) [inline cmds/bmp/task_command_test.go]:
```
It("executes without errors", ...)
```

File: integration/bmp/task_cmd_test.go [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+	Describe("$bmp task --task_id", func() {
+		Context("execute bmp task with default level", func() {
+			BeforeEach(func() {
+				session = RunBmp("task", "--task_id=1")
+			})
+
+			It("returns 0", func() {
+				Expect(session.ExitCode()).To(Equal(0))
+			})
```
Comment (maximilien) [inline integration/bmp/task_cmd_test.go]: Add checks on the output

File: cmds/bmp/task_command.go (hunk 1 above)
Comment (zhanggbj) [inline cmds/bmp/task_command.go]: Done

File: cmds/bmp/task_command_test.go (hunk 2 above)
Comment (zhanggbj) [inline cmds/bmp/task_command_test.go]: Done

File: cmds/bmp/task_command_test.go (hunk 3 above)
Comment (zhanggbj) [inline cmds/bmp/task_command_test.go]: Done

File: cmds/bmp/task_command_test.go (hunk 4 above)
Comment (zhanggbj) [inline cmds/bmp/task_command_test.go]: Done

File: integration/bmp/task_cmd_test.go (hunk 5 above)
Comment (maximilien) [inline integration/bmp/task_cmd_test.go]: Can you add the check? I'll merge as soon as you do

Comment (maximilien) [inline integration/bmp/task_cmd_test.go]: Something like: `Expect(session).To(Say("..."))`, of course replace `...` with an expected string in output

OR

`Expect(session.Wait().Out.Contents()).To(ContainSubstring("..."))`

Either should work fine.

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,

This is ready for review. (Some may need to update after PR #38 merged) Thanks.

**Pull request closed without merging**